Sources/XMTP/Client.swift

```diff
@@ -63,6 +63,8 @@ public class Client {
 	static var codecRegistry = {
 		var registry = CodecRegistry()
 		registry.register(codec: TextCodec())
+		registry.register(codec: AttachmentCodec())
```

Comment:
Missed this in the first pass, but I don't think we should default to enabling any additional codecs in the client. A new app developer should be able to make something that just supports text, and rely on the fallback of the content type to handle more complex things like attachments and remote attachments.

Reply:
I can get down with that. PR in https://github.com/xmtp/xmtp-ios/pull/72.